tests/lib/pkgdb.sh

```diff
@@ -317,7 +317,7 @@ distro_purge_package() {
 
     case "$SPREAD_SYSTEM" in
         ubuntu-*|debian-*)
-            quiet eatmydata apt-get remove -y --purge -y "$@"
+            eatmydata apt-get remove -y --purge -y "$@"
```

Comment:
Maybe a small comment that we can re-enable quiet once we have fixed the uc20 boot issues?

Reply:
Added